combat-trainer.lic

```diff
@@ -1912,6 +1931,14 @@ class GameState
     @aim_queue.shift
   end
 
+ def next_combo_action
```

Comment:
I was referring to `next_combo_action` and `use_barb_combos?`. They are only used internally to GameState.

The other methods you were looking at (with aim_queue.shift) are used externally.